infrastructure/gcp/keep-test/google-functions/keep-faucet/parse-account-address.js

```diff
@@ -3,18 +3,18 @@ const url = require('url')
 
 exports.parseAccountAddress = (request, response) => {
   requestUrl = url.parse(request.url, true)
-  if (requestUrl.query.address) {
-    return (address = requestUrl.query.address)
-  } else if (!/^(0x)?[0-9a-f]{40}$/i.test(address)) {
-    // check if it has the basic requirements of an address
+  if (requestUrl.query.account) {
+    return (account = requestUrl.query.account)
```

Comment:
`account` is never declared anywhere, and we never use it, right? I think you can just directly return here.